sdks/python/apache_beam/io/gcp/internal/clients/storage/storage_v1_client.py

```diff
@@ -20,10 +20,17 @@
 
 from __future__ import absolute_import
 
+import logging
+import time
```

Comment:
I believe this class is auto-generated. So this change might get reverted next time we update the client.

We should at least add a test that prevents this and move most of the changes here to another file (prob. everything other than "retry_func=xxx".

Reply:
Thanks for the comment. Updated.